layouts/partials/supported-countries.html

```diff
@@ -1,21 +1,21 @@
-<!-- Sooo, this is the easiest way *I* found to collect all countries with more than n records. If there's a better way, *please* implement that instead! -->
 {{ $.Scratch.Set "country_counts" (dict) }}
+
 {{ range where $.Site.Pages "Type" "company" }}
-{{ range .Param "relevant-countries" }}
-{{ if not (isset ($.Scratch.Get "country_counts") .) }}
-{{ $.Scratch.SetInMap "country_counts" . 1 }}
-{{ else }}
-{{ $.Scratch.SetInMap "country_counts" . (add (index ($.Scratch.Get "country_counts") .) 1) }}
-{{ end }}
-{{ end }}
+    {{ range .Param "relevant-countries" }}
+        {{ if not (isset ($.Scratch.Get "country_counts") .) }}
+            {{ $.Scratch.SetInMap "country_counts" . 1 }}
+        {{ else }}
+            {{ $.Scratch.SetInMap "country_counts" . (add (index ($.Scratch.Get "country_counts") .) 1) }}
+        {{ end }}
+    {{ end }}
```

Comment:
I wish there was a stylelint-esque tool for these templates. :D

Reply:
Oh yeah, I'd be happy even just with IDE support for them. :D